### ISSUE TITLE
alpha is not added in correctly

### ISSUE THREAD

**Issue by manyu90** [mkocabas/focal-loss-keras#6]
The role of alpha is to act as a weight between the two classes (pos and neg class have weights alpha,1-alpha respectively; so alpha should be chosen between 0 and 1)

In the current implementation, alpha is the same for both classes; so effectively does nothing to the loss function

**Comment by hwei-hw** [comment]
I have rewritten the code for fixing the alpha. You can find it in https://github.com/Atomwh/FocalLoss_Keras

**Comment by mkocabas** [comment]
I have corrected alpha value according to @Atomwh's implementation.
